docs/tutorials.md

```diff
@@ -0,0 +1,141 @@
+<!-- meta
+{
+  "node": {
+    "label": "Tutorials",
+    "category": {
+      "label": "Basics"
+    },
+    "metaData": {
+      "categoryPosition": 1,
+      "position": 3
+    }
+  }
+}
+meta -->
+
+# Tutorials
+
+A collection of advanced usecases and example implementations. If you are new to Luigi, you might want to have a look at our [**Getting Started**](getting-started.md) section first or one of our SPA implementations in the **Examples** section on the left.
+
+<!-- accordion:start -->
+
+### Use a SPA router for navigation and keep Luigi Core URL in sync
+
+It can be convenient to keep an existing routing strategy, and be able to use an existing microfrontend as drop-in without the need to refactor everyting to `LuigiClient.linkManager()`. 
+
+Concept wise it is neccessary to update the Luigi Core URL when routing internally with the microfrontend router, but without an update of the URL on Luigi Client side. This can be achieved by using the `linkManager()` [**withoutSync**](luigi-client-api.md#withoutsync) method.
+
+The following example shows how to configure the navigation node and use router events for automatic update of the url when using the microfrontends built in router.
+
+Be aware that this is a very simple example for navigation and special cases like modals, split views and other functionalities still require to use `LuigiClient`.
+
+For keeping the example simple, we use [**virtualTree**](navigation-parameters-reference.md#virtualtree) to allow any nested navigation. This is not mandatory, you can always specify the node tree by yourself and still use automatic navigation with router events.
+
+#### Luigi node configuration
+
+```javascript
+    {
+      pathSegment: 'Orders',
+      label: 'orders',
+      viewUrl: 'https://orders.microfrontend.tld/',
+      navigationContext: 'orders',
+      virtualTree: true
+    }
+```
+
+#### Use Angular Router for navigation
+
+Angular provides Router events. We are reacting on NavigationEnd to update the URL after a successful route change.
```

Comment:
```suggestion
Angular provides [Router events](https://angular.io/guide/router#router-events). We are reacting on `NavigationEnd` to update the URL after a successful route change.
```

Reply:
If you think the link is unnecessary here we can remove it. This is just how it's usually done in Kyma documentation 😅